src/css/styles.scss

```diff
@@ -191,7 +191,12 @@ header {
     margin: 1% 0%;
     padding: 1px 6px 5px 0px;
   }
-}
+
+  .cook-button {
```

Comment:
We may be able to format this button with our 'button-format' variable to match some styling of our other buttons